variables.tf

```diff
@@ -105,7 +105,7 @@ variable "resource_platform" {
 }
 
 variable "ssh_authorized_keys" {
-  #! Deprecation notice: Please use `ssh_public_key` or `ssh_public_key_path` instead
+  #! Deprecation notice: will be removed at next major release. Use `ssh_public_key` or `ssh_public_key_path` instead.
   description = "DEPRECATED: use ssh_public_key or ssh_public_key_path instead. Public SSH keys path to be included in the ~/.ssh/authorized_keys file for the default user on the instance."
   type        = string
```

Comment:
I'm a bit confused. Are you deprecating the plural form in favour of the singular?

Reply:
This is just rewording of the deprecation notice. The behavior is the same as in PR #68.
- `var.ssh_authorized_keys` was accepting only keys from file
- an issue was open asking to give more options for providing ssh key(s) for instances. #67

current behavior, since PR #68 (not part of a release yet):
- single key can be passed inline as a string -> use `var.ssh_public_key`
- multiple keys can be passed with a file -> use `ssh_public_key_path`.
- `ssh_authorized_keys` flagged for deprecation

There is currently no support to pass multiple keys directly as string.

I realize that the naming for these two new input variables are not the best, and we may look for a solution that could improve both naming and mechanism. I have to think if we can't manage it with a single variable ...

I suggest to take all your inputs regarding the ssh_keys and do another PR dedicated to it. Previous PR is not part of a release yet, so it is still good time to adjust.

On the same subject, #70 also needs to be addressed anyway.